.github/actions/e2e_test/action.yml

```diff
@@ -241,6 +241,16 @@ runs:
         existingConfig: ${{ steps.constellation-iam-create.outputs.existingConfig }}
         artifactNameSuffix: ${{ steps.create-prefix.outputs.prefix }}
 
+    - name: Deploy logcollection
+      id: deploy-logcollection
+      # TODO(msanft):temporarily deploy in debug clusters too to resolve "missing logs"-bug
+      # see https://dev.azure.com/Edgeless/Edgeless/_workitems/edit/3227
+      # if: inputs.isDebugImage == 'false'
```

Comment:
the reason for keeping the old debug log collection is to collect them even before the cluster is running?
I.e. logstash only collects new events that it sees and not prior ones? If this was the case, how could this help us see how the debug filebeat container failed?

Reply:
Filebeat collects systemd-logs (which should include why the podman container from debugd fails) - this will help us resolve the bug, hopefully

Reply:
Yes. Filebeat also collects logs made prior to its deployment